modules/related-posts/jetpack-related-posts.php

```diff
@@ -228,13 +262,11 @@ public function get_target_html_unsupported() {
 		}
 		return "\n\n<!-- Jetpack Related Posts is not supported in this context. -->\n\n";
 	}
-
 	/**
 	 * ========================
 	 * PUBLIC UTILITY FUNCTIONS
 	 * ========================
 	 */
-
```

Comment:
Is there a benefit in removing this line break and make this and the next PHPdoc block a single block?